### DISCUSSION TITLE
How to keep updated my local installation?

### DISCUSSION THREAD

**Discussion by ProgrammingDinosaur** [hlky/stable-diffusion#179] [Q&A] [answered]
Sorry if this comes as a newby question. Title says it all, should i clone the whole repo again when it's updated? Or just the environment.yaml file? Any help is appreciated, thanks

**Comment by DenShlk** [comment, marked as answer]
you can use `git pull` to update. you can simply do this once in a while, or check with this repo to see when there are new updates.
If you are making changes to files in repository you'll have to merge them with changes from authors, in this case i recommend using ide.
